src/java.xml/share/classes/module-info.java

```diff
@@ -27,6 +27,231 @@
+ * implementation specific features and properties. This section describes the
+ * naming convention, System Properties, jaxp.properties, scope and order,
+ * and processors to which a property applies. A table listing the implementation
+ * specific features and properties which the implementation currently supports
+ * can be found at the end of this note.
+ *
+ * <h3>Naming Convention</h3>
+ * The names of the features and properties are fully qualified, composed of a
+ * prefix and name.
+ * <p>
+ * <h4>Prefix</h4>
+ * The prefix for JDK properties is defined as:
+ * <pre>
+ *     {@code http://www.oracle.com/xml/jaxp/properties/}
+ * </pre>
+ *
+ * The prefix for features:
+ * <pre>
+ *     {@code http://www.oracle.com/xml/jaxp/features/}
+ * </pre>
+ *
+ * The prefix for System Properties:
+ * <pre>
+ *     {@systemProperty jdk.xml.}
+ * </pre>
+ * <p>
+ * <h4>Name</h4>
+ * A name may consist of one or multiple words that are case-sensitive.
+ * All letters of the first word shall be in lowercase, while the first letter of
+ * each subsequent word capitalized.
+ * <p>
+ * An example of a property that indicates whether an XML document is standalone
+ * would thus have a format:
+ * <pre>
+ *     {@code http://www.oracle.com/xml/jaxp/properties/isStandalone}
+ * </pre>
+ * and a corresponding System Property:
+ * <pre>
+ *     {@systemProperty jdk.xml.isStandalone}
+ * </pre>
+ *
+ * <h3>System Properties</h3>
+ * A property may have a corresponding System Property that has the same name
+ * except the prefix as shown above. A System Property should be set prior to
+ * the creation of a processor and may be cleared afterwards.
+ *
+ * <h3>jaxp.properties</h3>
+ * A system property can be specified in the {@code jaxp.properties} file to
+ * set the behavior for all invocations of the JDK or JRE. The format is
+ * {@code system-property-name=value}. For example:
+ * <pre>
+ *     {@code jdk.xml.isStandalone=true}
+ * </pre>
+ *
+ * <h3 id="ScopeAndOrder">Scope and Order</h3>
+ * The {@link javax.xml.XMLConstants#FEATURE_SECURE_PROCESSING} feature
+ * (hereafter referred to as FSP) is required for XML processors including DOM,
+ * SAX, Schema Validation, XSLT, and XPath. Any properties flagged as
+ * {@code "security: yes"} (hereafter referred to as security properties) in table
+ * <a href="#FeaturesAndProperties">Features And Properties</a>
+ * are enforced when FSP is set to true. Such enforcement includes setting security
+ * features to true and limits to the defined values shown in the table.
+ * The property values will not be affected, however, when setting FSP to false.
+ * <p>
+ * When the Java Security Manager is present, FSP is set to true and can not be
+ * turned off. The security properties are therefore enforced.
+ * <p>
+ * Properties specified in the jaxp.properties file affect all invocations of
+ * the JDK and JRE, and will override their default values, or those that may
```

Comment:
We've phased out the "JRE", so it can be dropped here.

Reply:
Removed all cases.